landmarkerio/__init__.py

```diff
@@ -3,7 +3,7 @@
 
 class FileExt(Enum):
     lm = '.ljson'
-    template = '.txt'
+    template = '.yml'
```

Comment:
Yep it changes the logic completely, the code is still there to be able to do it but it doesn't load them by default.